lib/index.js

```diff
@@ -43,15 +43,29 @@ app.use(session({
   saveUninitialized: false,
 }));
 
+const oidc = new ExpressOIDC({
+  issuer: config.oidc.issuer,
+  client_id: config.oidc.clientId,
+  client_secret: config.oidc.clientSecret,
+  redirect_uri: config.oidc.redirectUri,
+  scope: 'openid profile email'
+});
+
+// Let ExpressOIDC add the /login and /authorization-code/callback routes
+app.use(oidc.router);
+
 // These are the routes that need to be implemented to handle the
 // authorization code scenarios
 app.get('/', handlers.scenarios);
 app.get('/authorization-code/login-redirect', handlers.loginRedirect);
-app.get('/authorization-code/login-custom', handlers.loginCustom);
```

Comment:
In this case we still don't have an option for login-custom right?

Reply:
Yes, we don't have the custom login option yet, but I think we need to discuss how do we handle it. Right now, I just removed the route handler. But the front end still has a "login-custom" option. Do we remove it from the front end as well, or just display a "functionality coming soon" page?

Reply:
Let's remove this login-custom link from the front-end as well, for now.  We can add this back in when we apply the samples V2 strategy to this repo.